src/app/calculator/process-cooling/cooling-tower-basin/cooling-tower-basin.component.html

```diff
@@ -57,8 +57,18 @@ <h3>Weather Data</h3>
 				</a>
 			</div>
 		</div>
-		<div *ngIf="!displayWeatherTab" class="d-flex mr-2 ml-2">
+	
+
+			<div *ngIf="!displayWeatherTab" class="d-flex mr-2 ml-2">
 			<div class="d-flex flex-column w-100">
+				<div class="mb-3 d-block" *ngIf="tabSelect == 'results'">
+					<div class="w-100 justify-content-center btn-group btn-group-sm results pt-1 pb-1">
+						  <button type="button" class="btn btn-secondary btn-sm" (click)="toggleWeatherResults(true)"
```

Comment:
Choosing weather results should be disabled when there is no weather data. Add `						  [disabled]="!hasWeatherBinsData"
`